src/lib/searchUtilActions/search.ts

```diff
@@ -92,7 +66,7 @@ export async function handleAasRegistrySearch(
  * @param {string} searchAasId - The AAS ID to resolve using the discovery service.
  * @returns {Promise<string | null>} A promise that resolves to the resolved AAS ID as a string, or `null` if the AAS ID is not found.
  */
-export async function handleAasDiscoverySearch(searchAasId: string): Promise<string | null> {
+export async function handleAasDiscoverySearch(searchAasId: string): Promise<string[] | null> {
```

Comment:
I think this might be a copy paste error, the discovery takes an asset id not an aas id